apps/admin/backend/src/app.cvr_files.test.ts

```diff
@@ -621,59 +598,8 @@ test('error if cast vote records from different files share same ballot id but h
 
   expect(result.err()).toMatchObject({
     type: 'ballot-id-already-exists-with-different-data',
-    message:
-      'Found cast vote record at index 0 that has the same ballot id as a previously imported cast vote record, but with different data.',
   });
 
   expect(await apiClient.getCastVoteRecordFiles()).toHaveLength(1);
   await expectCastVoteRecordCount(apiClient, 184);
 });
-
-test('error if a layout is invalid', async () => {
```

Comment:
This test case is still relevant. But requires some work to test with the new CVR format. I'm going to work on gaps in testing / getting back up to 100% code coverage in a subsequent PR.